# Array of detected lane lines in a single frame
Header header
LaneLine[] lines
